CRM/Emailfiling/Hook/BuildForm/AddSettingOutbound.php

```diff
@@ -46,8 +48,8 @@ private function shouldRun($formName) {
    *   Form Class object.
    */
   private function addField(CRM_Core_Form &$form) {
-    $fieldName = 'emailfilingIsOutboundProcessingEnabled';
-    $fieldLabel = ts('Store a copy of sent emails on activity');
+    $fieldName = EmailfilingConst::settingOutbound('name');
+    $fieldLabel = EmailfilingConst::settingOutbound('title');
```

Comment:
Why not fetch the whole outbound setting i.e `EmailfilingConst::settingOutbound()` in the extension constructor, store this in a variable and use it throughout the class instead of invoking the function all the time.